server/controller/routes/user/getUser.js

```diff
@@ -0,0 +1,13 @@
+const { getUserData } = require('../../../database/queries/user');
+
+const getUser = async (req, res, next) => {
+  try {
+    const { userId } = req;
+    const { rows } = await getUserData(userId);
+    return res.json({ data: rows[0] });
```

Comment:
What will happen if a userId that doesn't exist was requested here? I would think that if `rows` is empty it would make sense to return a 404 to the client

Reply:
Hi jake, 

Thank you for taking the time to review our project

this user Id we get from token after login process , and this route is protected it's impossible to get 404

Reply:
Hi jake, 

Thank you for taking the time to review our project

this user Id we get from token after login process , and this route is protected it's impossible to get 404, is just for user profile